pages/dev/axelarjs-sdk/token-transfer-dep-addr.md

```diff
@@ -81,55 +81,169 @@ async function main() {
 main();
 ```
 
-### 5. Generate a deposit address using the SDK
+### 5. Generate a deposit address
 
-Call `getDepositAddress`:
+> When making your deposit, please ensure that the amount is greater than the cross-chain relayer gas fee. The relayer gas fee can be calculated with `getTransferFee` function above.
+
+The `AxelarAssetTransfer` class exposes the `getDepositAddress` function. With this function, you can create a **deposit address**. Here is the function signature 👇
 
 ```tsx
-async getDepositAddress(
-  fromChain: string, // source chain
-  toChain: string, // destination chain
-  destinationAddress: string, // destination address to transfer the token to
-  asset: string, // denom of asset. See note (2) below
+interface GetDepositAddressParams {
+  fromChain: string; // See note (3) below
+  toChain: string; // See note (3) below
+  destinationAddress: string; // See note (1) below
+  asset: string; // denom of asset. See note (2) below
   options?: {
-    _traceId: string;
-  }
-): Promise<string> {}
+    shouldUnwrapIntoNative?: boolean;
+    refundAddress?: string; // optional. See note (4) below
+  };
+}
+
+async getDepositAddress({
+  fromChain, toChain, destinationAddress, asset
+}: GetDepositAddressParams): Promise<string> {}
+```
+
+#### EXAMPLES
+All possible use cases are covered in the scenarios below
+
+##### 1. Send tokens from a Cosmos-based chain
+
+Transfer any Axelar-supported asset from a cosmos-based chain to a destination chain. 
+* Destination chains can be either an EVM chain or cosmos-based chain
+* When sending to an EVM chain, the asset will arrive as an ERC-20 asset
+
+Example acceptance criteria: 
+`I want to send axlUSDC on Osmosis and receive the equivalent ERC-20 version on Avalanche.`
+
+```tsx
+const sdk = new AxelarAssetTransfer({ environment: "testnet" });
+
+const fromChain = "osmosis-5", 
```

Comment:
Could we use the CHAINS config object? (request from Milap)

```ts
const fromChain = CHAINS.TESTNET.OSMOSIS;
const toCHain = CHAINS.TESTNET.AVALANCHE;
```

Reply:
done